app/src/main/java/com/github/se/eventradar/model/event/Event.kt

```diff
@@ -11,13 +11,19 @@ enum class EventCategory {
   COMMUNITY
 }
 
+data class Ticket(val name: String, val price: Double, val quantity: Int)
+// new event struct final
 data class Event(
-    val id: String,
-    val hostUserId: String,
-    val name: String,
+    val eventName: String,
+    val eventPhoto: String,
+    val start: LocalDateTime,
+    val end: LocalDateTime,
+    val location: Location,
     val description: String,
-    val date: LocalDate,
-    val time: LocalTime,
-    val category: EventCategory
-    // TODO: Add 'location' attribute once the Map feature is implemented
+    val ticket: Ticket,
+    val contact: String,
+    val organiserList: Set<String>,
+    val attendeeList: Set<String>,
```

Comment:
I read in the blueprint that all coorganizer can edit the events they coorganize, can they as well delete it ?
if yes we can keep it like that, else we may require a "creator" value.

the type is a Set of string. Does it contains the users uid ? Could you specify it with a comment ?

Reply:
I agree that this is an important distinction to figure out now. I also think it would make more sense to save the reference to the host's user ID in the User Database than just to have their contact email.

Reply:
For now, let us keep it this way, if we wanted to make the difference then we should implement the employee list and organiser list but decided not to and to keep it as an extra.
String in the list will be the uids of the user in database only :)

Reply:
Which means all users in the organisers list got the same right to edit and delete the event ?